picard/webservice.py

```diff
@@ -484,30 +486,55 @@ def _find(self, entitytype, handler, kwargs):
             value = QUrl.toPercentEncoding(unicode(value))
             queryargs[str(name)] = value
         path = "/ws/2/%s" % (entitytype)
-        return self.get(host, port, path, handler, queryargs=queryargs)
-
-    def find_releases(self, handler, **kwargs):
-        return self._find('release', handler, kwargs)
-
-    def find_tracks(self, handler, **kwargs):
-        return self._find('recording', handler, kwargs)
-
-    def find_artists(self, handler, **kwargs):
-        return self._find('artist', handler, kwargs)
-
-    def _browse(self, entitytype, handler, kwargs, inc=[], priority=False, important=False):
+        return self.get(host, port, path, handler, queryargs=queryargs,
+                xml=xml, priority=priority, important=important, mblogin=mblogin,
+                cacheloadcontrol=cacheloadcontrol, refresh=refresh)
+
+    def find_releases(self, handler,
+            xml=True, priority=False, important=False, mblogin=False,
```

Comment:
The repitition of keyword arguments in the new code looks unnecessary, why not just pass `**kwargs` through?